media-libs/libplacebo/libplacebo-2.43.0-r1.ebuild

```diff
@@ -32,6 +32,10 @@ BDEPEND="virtual/pkgconfig"
 
 RESTRICT="!test? ( test )"
 
+PATCHES=(
+	"${FILESDIR}"/${PN}-vulkan-headers-compatibility.patch
```

Comment:
- We usually prefix with `${P}` not `${PN}`
- Compatibility with what version of vulkan headers - would it be useful to include in the file name?